common/changes/@itwin/ec3-widget-react/uyen-ec3-widget-migration_2023-08-02-20-20.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/ec3-widget-react",
+      "comment": "Migrated UI items for iTwin UI 2.x and iTwinJS 4.x upgrade.",
+      "type": "minor"
```

Comment:
Bumping a peer dependency to a major version is usually considered a breaking/major change.

Reply:
I just changed it to 'major'.

Reply:
Hey Ben,
We understand that a major version change is a significant step, but in this case, we believe a minor bump is appropriate for this widget, given its early stage of development.

Reply:
Ah didn't see the widget was at version 0.2.0 - yep that makes sense. Sorry for the confusion!